beacon-chain/blockchain/forkchoice/process_attestation.go

```diff
@@ -249,21 +249,23 @@ func (s *Store) updateAttVotes(
 	tgtRoot []byte,
 	tgtEpoch uint64) error {
 
-	for _, i := range append(indexedAtt.CustodyBit_0Indices, indexedAtt.CustodyBit_1Indices...) {
+	indices := append(indexedAtt.CustodyBit_0Indices, indexedAtt.CustodyBit_1Indices...)
+	newVoteIndices := make([]uint64, 0)
```

Comment:
can you make the capacity for both these slices `len(indices)`

Reply:
We don't know the size, because in the for loop there is a conditional:
```go
newVotes := make([]*pb.ValidatorLatestVote, 0)
	for _, i := range indices {
		vote, err := s.db.ValidatorLatestVote(ctx, i)
		if err != nil {
			return errors.Wrapf(err, "could not get latest vote for validator %d", i)
		}
		if vote == nil || tgtEpoch > vote.Epoch {
			newVotes = append(newVotes, &pb.ValidatorLatestVote{
				Epoch: tgtEpoch,
				Root:  tgtRoot,
			})
			newVoteIndices = append(newVoteIndices, i)
		}
```

Reply:
we dont know the size, but we know what could be the maximum possible size. Which would be `len(indices)`, otherwise we would be allocating a new array whenever we append a single element in each iteration, since the slice starts with a capacity of 0